packages/sdk/src/PreVerificationGasCalculator.ts

```diff
@@ -41,13 +67,34 @@ export interface PreVerificationGasCalculatorConfig {
   readonly estimationPaymasterDataSize: number
 }
 
+export interface GasOptions {
+  /**
+   * if set, assume this gas is actually used by verification of the UserOperation.
+   * (as checked during UserOperation simulation)
+   */
+  verificationGas?: number
```

Comment:
```suggestion
  verificationGasUsed?: number
```

Reply:
Your interface literally has two fields: `totalGasUsed` and `verificationGas`. Both are "gas used" values. Naming consistency is a good thing in my opinion.